libs/langchain-mongodb/langchain_mongodb/graphrag/prompts.py

```diff
@@ -46,6 +46,37 @@
 Instead of using specific and momentary types such as 'worked_at', use more general and timeless relationship types
 like 'employee'. Add details as attributes. Make sure to use general and timeless relationship types!
 
+### CRITICAL: Array Length Alignment
+The relationships object contains three arrays: `target_ids`, `types`, and `attributes`.
+**These three arrays MUST have EXACTLY the same length.**
+- Each position (index) in these arrays describes ONE complete relationship.
+- Position 0 in `target_ids`, `types`, and `attributes` together describe the first relationship.
+- Position 1 in `target_ids`, `types`, and `attributes` together describe the second relationship.
+- And so on...
+
+If a relationship has no attributes, you MUST still include an empty object `{{}}` in the `attributes` array at that position.
+
+Example of CORRECT alignment:
+```json
+"relationships": {{
+  "target_ids": ["Entity A", "Entity B"],
+  "types": ["partners", "supplier"],
+  "attributes": [
+    {{"since": ["2020"]}},
+    {{}}
+  ]
+}}
+```
+
+Example of INCORRECT (DO NOT DO THIS):
+```json
+"relationships": {{
+  "target_ids": ["Entity A", "Entity B"],
+  "types": ["partners"],
+  "attributes": [{{"since": ["2020"]}}]
+}}
+```
+
```

Comment:
This additional docstring feels separate & apart from the goal of just graph visualizations. This should be split out into a different PR.